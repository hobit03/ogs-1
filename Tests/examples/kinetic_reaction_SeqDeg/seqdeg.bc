#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE RIGHT_BOUNDARY
 $DIS_TYPE
  CONSTANT 0.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  so4
 $GEO_TYPE
  POLYLINE LEFT_BOUNDARY
 $DIS_TYPE
  CONSTANT  4.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  o2
 $GEO_TYPE
  POLYLINE LEFT_BOUNDARY
 $DIS_TYPE
  CONSTANT  0.26
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  tracer
 $GEO_TYPE
  POLYLINE INJECTION_LINE
 $DIS_TYPE
  CONSTANT  1.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  c12h1
 $GEO_TYPE
  POLYLINE INJECTION_LINE
 $DIS_TYPE
  CONSTANT  1.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  tracer
 $GEO_TYPE
  POLYLINE NOTINJECTION_LINE
 $DIS_TYPE
  CONSTANT  0.0
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  c12h1
 $GEO_TYPE
  POLYLINE NOTINJECTION_LINE
 $DIS_TYPE
  CONSTANT  0.0
#STOP
