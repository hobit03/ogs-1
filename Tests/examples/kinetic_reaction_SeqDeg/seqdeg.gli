GeoSys-BC: Boundary Conditions ------------------------------------------------
#POINTS
0  0.0   0.0 0.0 0.0 0.0 0.0 0.0
1  0.0  0.0 5.0 0.0 0.0 0.0 0.0
2  400.0  0.0 0.0 0.0 0.0 0.0 0.0
3  400.0  0.0 5.0 0.0 0.0 0.0 0.0
4  0.0  0.0 3.0 0.0 0.0 0.0 0.0
5  0.0  0.0 5.0 0.0 0.0 0.0 0.0
6  0.0  0.0 5.0 0.0 0.0 0.0 0.0
7  400.0  0.0 5.0 0.0 0.0 0.0 0.0
8  0.0  0.0 0.0 0.0 0.0 0.0 0.0
9  0.0  0.0 5.0 0.0 0.0 0.0 0.0
10  0.0  0.0 2.9 0.0 0.0 0.0 0.0
#POLYLINE
 $NAME 
  LEFT_BOUNDARY
 $EPSILON
  1.0e-4
 $POINTS
  0
  1
#POLYLINE
 $NAME 
  RIGHT_BOUNDARY
 $EPSILON
  1.0e-4
 $POINTS
  2
  3
#POLYLINE
 $NAME 
  INJECTION_LINE
 $EPSILON
  1.0e-4
 $POINTS
  4
  5
#POLYLINE
 $NAME 
  NOTINJECTION_LINE
 $EPSILON
  1.0e-4
 $POINTS
  0
  10 
#POLYLINE
 $NAME 
  X_LINE
 $EPSILON
  1.0e-4
 $POINTS
  6
  7
#POLYLINE
 $NAME 
  Y_LINE
 $EPSILON
  1.0e-4
 $POINTS
  8
  9
#STOP
