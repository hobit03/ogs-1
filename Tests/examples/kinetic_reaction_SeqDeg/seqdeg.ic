#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  tracer
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  c12h1
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  so4
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#INITIAL_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  o2
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1.0e-20
#STOP
