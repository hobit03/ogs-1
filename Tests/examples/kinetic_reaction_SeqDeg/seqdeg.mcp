#COMPONENT_PROPERTIES  ; comp1
 $NAME
  tracer ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp2
 $NAME
  c12h1 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp3
 $NAME
  so4 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#COMPONENT_PROPERTIES  ; comp4
 $NAME
  o2 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant
#STOP
