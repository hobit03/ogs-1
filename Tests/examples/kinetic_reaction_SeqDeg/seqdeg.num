#NUMERICS
 $PCS_TYPE
  GROUNDWATER_FLOW
 $ELE_GAUSS_POINTS
  3
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  2      6 1.e-010       5000           1.0   100       4
#NUMERICS
 $PCS_TYPE
  KIN_REACT_GIA
 $ELE_GAUSS_POINTS
  3
 $LINEAR_SOLVER
; method error_tolerance max_iterations theta precond storage
  2      6 1.e-010       5000           1.0   100       2
#STOP
