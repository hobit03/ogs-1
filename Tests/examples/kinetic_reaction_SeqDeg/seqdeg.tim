GeoSys-TIM: Timestepping ------------------------------------------------
#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   7500  24
 $TIME_END
  180000
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  KIN_REACT_GIA
 $TIME_STEPS
   7500  24
 $TIME_END
  180000
 $TIME_START
  0.0
#STOP
