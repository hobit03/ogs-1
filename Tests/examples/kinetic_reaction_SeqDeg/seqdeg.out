#OUTPUT // profile
 $NOD_VALUES
  HEAD
  Tracer
  CH2O
  SO42-
  O2
  CO2
  HCO3-
  H2S
  Biomass_oxygen
  Biomass_sulfate
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 100
#STOP




#OUTPUT // profile
 $NOD_VALUES
  HEAD
  tracer
  c12h1
  c13h1
  c12h2
  so4
  o2
 $GEO_TYPE
  POLYLINE X_LINE
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 10
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  tracer
  c12h1
  c13h1
  c12h2
  so4
  o2
 $GEO_TYPE
  POLYLINE Y_LINE
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 10
#STOP
